covid_model/fit_disconnected_co_regions.py

```diff
@@ -0,0 +1,103 @@
+import matplotlib.pyplot as plt
```

Comment:
Fits each regional model, allows parallel processing if specified. writes specification results to database serially to avoid concurrency issues.

Reply:
Super nitpicky comment: I've been following a convention where any scripts that are runnable are named "run_*.py". Open to changing that convention, but it helps me keep thing straight when I'm looking through the repo.

Reply:
No this makes sense, I don't mind changing script names. Also applies to "disconnected_to_conencted_betas_co_regions.py". Should that script be in "analysis"?